bin/zopen-importenvs

```diff
@@ -1,18 +1,96 @@
-#!/bin/sh
-#
-# Used for development to source all the boot .envs
-# . importenvs
-
-if [ ! -d "${HOME}/zopen/boot/" ]; then
-  echo "boot directory does not exist.  Existing"
-  exit 4;
+#!/bin/env bash
+# bash is required because BASH_SOURCE is used here to locate the sourced script 
+# Used for development to source all the boot/prod .envs
+
+
+(return 0 2>/dev/null) && sourced=1 
+# What exit code did that give?
+if [[ $sourced -ne 1 ]]
+then
+    echo "This script is not sourced."
+    echo "USAGE: . ./zopen-importenvs <path to buildenv to fetch dependency>"
+    exit 6     
```

Comment:
minor: exit 6 seems a bit random to me.  

future consideration:  We should probably come up with a consistent set of error codes (maybe define in common.inc)

Reply:
I have modified it.

Reply:
Also, i have added the exit 1, condition in another error case too. As of now i have used exit 1 in 2 use cases in this script.